easybuild/easyconfigs/n/NVHPC/NVHPC-20.7-GCC-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'NVHPC'
+version = '20.7'
+local_gccver = '9.3.0'
+versionsuffix = '-GCC-%s' % local_gccver
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
+
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.34', '', ('GCCcore', local_gccver)),
+    ('CUDA', '11.0', '', SYSTEM),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.13', '', SYSTEM)
+]
+
+default_cuda_version = "11.0"
```

Comment:
You can use the `%(cudaver)s` template here:

```python
default_cuda_version = "%(cudaver)s"

Reply:
CUDA is not a hard dependency (see the discussion in the EasyBlock); so I'd rather be explicit here.

Reply:
This should be changed to `default_cuda_version = None` to rely on what the NVHPC easyblock does by default (use the CUDA version listed as a dep unless told otherwise)?

Reply:
Yep, that's in the current file.